openedx/adg/admin_dashboard/admin_settings/tests/test_views.py

```diff
@@ -13,39 +13,70 @@
 from openedx.core.djangolib.testing.utils import skip_unless_lms
 
 
+@pytest.fixture(name='admin_accounts_list_request')
+@skip_unless_lms
+def admin_account_list_view_request(adg_admin):
+    """
+    Fixture which returns ADG admin user.
+    """
+    url = reverse('admin_accounts')
+    admin_accounts_get_request = RequestFactory().get(url)
```

Comment:
We have a fixture for request_factory in the conftest, I believe. Let's use that here